modules/heat_conduction/src/materials/SideSetHeatTransferMaterial.C

```diff
@@ -0,0 +1,125 @@
+/****************************************************************/
+/*               DO NOT MODIFY THIS HEADER                      */
+/*  MAMMOTH: - Radiation Kinetics and Reactor Physics Software  */
+/*                                                              */
+/*           (c) 2016 Battelle Energy Alliance, LLC             */
+/*                   ALL RIGHTS RESERVED                        */
+/*                                                              */
+/*          Prepared by Battelle Energy Alliance, LLC           */
+/*            Under Contract No. DE-AC07-05ID14517              */
+/*            With the U. S. Department of Energy               */
+/*                                                              */
+/*            See COPYRIGHT for full restrictions               */
+/****************************************************************/
+
+#include "SideSetHeatTransferMaterial.h"
+
+// Stefan - Boltzmann Constant
+#define SIGMA 5.670374419E-8
```

Comment:
Minor: We typically use C++ style constants, not CPP style.

Reply:
I assume you mean replacing this line with:
```
const Real SIGMA = 5.670374419E-8;
```

Reply:
Yep